R/rbabylon.R

```diff
@@ -47,7 +60,7 @@ bbi_exec <- function(.cmd_args, .dir = ".", .verbose = FALSE, .wait = FALSE, ...
   res <- list()
   res[[PROC_PROCESS]] <- p
   res[[PROC_STDOUT]] <- output
-  res[[PROC_BBI]] <- bbi_exe_path
+  res[[PROC_BBI]] <- .bbi_exe_path
```

Comment:
oops, were we not testing this?

Reply:
This PR adds `.bbi_exe_path` as an argument to `bbi_exec()`. The existing implementation calls `getOption()` within `bbi_exec()` and assigns the result to `bbi_exe_path`. By making the path an argument, we no longer need the internal `getOption()` call. The package convention prefixes argument names with `.`, so this assignment should be for the argument `.bbi_exe_path` rather than the internal variable `bbi_exe_path`.

Reply:
ah ok. I follow.